bumble/smp.py

```diff
@@ -525,6 +525,13 @@ async def compare_numbers(self, number: int, digits: int) -> bool:
     async def get_number(self) -> int:
         return 0
 
+    async def get_string(self, min_length=0, max_length=-1) -> Optional[str]:
```

Comment:
I'm not sure about `min_length`, @barbibulle what do you think ?

Reply:
I suggest to:
- remove `min_length` since `None` is available
- remove default value for `max_length`
- return `None` by default
As follow:
```python
async def get_string(self, max_length) -> Optional[str]:
    '''Returns a string of max_length bytes.'''
    return None
```

Reply:
To be precise, the comment should probably say: """Returns a string whose utf-8 encoding is up to max_length bytes."""